auxiliary.py

```diff
@@ -5,11 +5,11 @@
 
 
 uco = mpatches.Patch(color=(1., 1., 0.), label='UCO')
-buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer')
+buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer layer')
 pyc = mpatches.Patch(color=(1., 0., 0.), label='PyC')
 sic = mpatches.Patch(color=(0.6, 0.3, 0.), label='SiC')
-matrix = mpatches.Patch(color=(0.63, 0.63, 0.63), label='Matrix')
-block = mpatches.Patch(color=(0.61, 1., 0.91), label='Block')
+matrix = mpatches.Patch(color=(0.63, 0.63, 0.63), label='Compact matrix')
+block = mpatches.Patch(color=(0.61, 1., 0.91), label='Fuel Block')
```

Comment:
Is this the graphite surrounding the fuel? If so, it's not fuel, but rather H451 Graphite. Perhaps "Graphite Fuel Block"?
```suggestion
block = mpatches.Patch(color=(0.61, 1., 0.91), label='Graphite Fuel Block')
```

Reply:
Yes, it is H451 Graphite. I called it 'Fuel Block' because that is how it is called in the OECD-NEA report. 

![image](https://user-images.githubusercontent.com/43791150/85464916-cdfa0a00-b56d-11ea-9c57-3ec96512239f.png)

How about 'Fuel Block Graphite'?

Reply:
Fuel Block (Graphite) makes grammatical sense, but "Fuel Block Graphite" does not. As you know, the ordering of adjectives in English is a bit odd, making "Graphite Fuel Block" clear where "Fuel Block Graphite" is a bit confusing. Either "Fuel Block (Graphite)" or "Graphite Fuel Block" is ok with me.

Reply:
Okay!